apps/are_form/behaviours/referencelist.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const staticExclusionDates = require('../lib/staticExclusionDates');
+const staticAppealStages = require('../lib/staticAppealStages');
+const moment = require('moment');
+const dateFormat = 'dddd DD MMMM YYYY';
+
+module.exports = superclass => class ReferenceList extends superclass {
+    getValues(req, res, callback) {
+        super.getValues(req, res, (err, values) => {
+            let json = req.sessionModel.toJSON();
+
+            if (req.url === '/appealstages') {
+                json['reference-appeal-list'] = [].concat(staticAppealStages.getstaticAppealStages());
+                json['reference-appeal-list-count'] = json['reference-appeal-list'].length;
+            } else if (req.url === '/exclusiondates') {
+                json['reference-exclusiondate-list-england-and-wales'] =
+                    [].concat(staticExclusionDates.getExclusionDays('England & Wales'));
+                json['reference-exclusiondate-list-scotland'] =
+                    [].concat(staticExclusionDates.getExclusionDays('Scotland'));
+                json['reference-exclusiondate-list-northern-ireland'] =
+                    [].concat(staticExclusionDates.getExclusionDays('Northern Ireland'));
```

Comment:
don't like the spacing with `Northern Ireland` this is bad practice but I can understand it might not be something you did

Reply:
Same as for England & Wales, the field name has been used.

Reply:
keep for now too much to refactor